src/schema/v2/artworksForUser.ts

```diff
@@ -47,33 +52,83 @@ export const getArtistAffinities = async (
   return artistIds
 }
 
+export const getAffinityArtworks = async (
+  artistIds: string[],
+  gravityArgs,
+  context: ResolverContext
+): Promise<any[]> => {
+  if (artistIds.length === 0) return []
+
+  const { size, offset } = gravityArgs
+  const { artworksLoader } = context
+
+  const artworkParams = {
+    artist_ids: artistIds,
+    availability: "for sale",
+    offset,
+    size,
+    sort: "-published_at",
+  }
+
+  const body = await artworksLoader(artworkParams)
+
+  return body
+}
+
+export const getBackfillArtworks = async (
+  remainingSize: number,
+  includeBackfill: boolean,
+  context: ResolverContext
+): Promise<any[]> => {
+  if (!includeBackfill || remainingSize < 1) return []
+
+  const { artworksLoader, setItemsLoader } = context
+
+  // should this be injected in the env? i could also send yet another api call
+  // to get this id via the key of artwork-backfill. could also consider
+  // caching this...
+  const backfillSetId = "SHHH"
```

Comment:
I'm interested in ideas here. What I really wish I could do is pass `artwork-backfill` like a slug but the Gravity API does not support this. I guess I should inject this using ENV vars, right?

Reply:
It feels weird to pass things along as an ENV var, but we do do it elsewhere. I think if anything we should update the  gravity endpoint to accept a new optional arg (if it makes sense) first.

Reply:
More chatter here but I ended up going with an ENV var:

https://artsy.slack.com/archives/CP9P4KR35/p1652907794787929

Reply:
Update: I did not go with an ENV var haha. I was able to figure out that I could call the setsLoader with the key of `artwork-backfill` and get the id. Those two calls are not authenticated so that means they are cached. 😎